src/sources/forus-platform/pug/tpl/printables/printable-voucher-qr_code.pug

```diff
@@ -1,10 +1,40 @@
 //- Printable class is required 
 .printable.printable-voucher-qr_code
-    .printable-title(ng-if="$ctrl.voucher.type != 'product'") Uw {{ $ctrl.voucher.fund.name }}-voucher
-    .printable-title(ng-if="$ctrl.voucher.type == 'product'") Uw {{ $ctrl.voucher.product.name }}-voucher
-    br
-    qr-code(qr-value="$ctrl.qrCodeValue" qr-type="voucher")
-    .printable-description De verloopdatum van deze voucher is {{ $ctrl.voucher.expire_at_locale }}
+    .printable-main-info
+        .printable-usage-info 
+            span(i18n="voucher_printable.{{$ctrl.voucherType}}_voucher.labels.use_for")
+            br
+            strong {{ $ctrl.voucher.expire_at_locale }}
+
+        .printable-logo
+            img(ng-src="./assets/img/implementation-logo.png")    
+    
+        qr-code(qr-value="$ctrl.qrCodeValue" qr-type="voucher")
+        .printable-title {{ $ctrl.printableTitle }}
+        .printable-description(
+            i18n="voucher_printable.{{$ctrl.voucherType}}_voucher.labels.description" 
+            i18n-vars="{ printableTitle: $ctrl.printableTitle }")
+
+    .printable-additional-info
+        .description(i18n="voucher_printable.{{$ctrl.voucherType}}_voucher.labels.show_to")  
+        .reference {{ $ctrl.voucherType != 'product' ? $ctrl.organization.site : $ctrl.organization.name }} 
```

Comment:
$ctrl.organization.site is empty on pdf and not displayed 
![image](https://user-images.githubusercontent.com/10818702/69963307-433db900-1510-11ea-83f8-1ea843dae8d1.png)

Reply:
Should that block be hidden (for budget vouchers) if the organization doesnt have a website?

Reply:
Ah you don't understand which website should be displayed. Its the implementations webshop.
The text above is the dutch translation for "Show this voucher to providers that are approved, you can find this list on:"

Reply:
It should display the webshop_url in implementation table (or general url if no implementation applicable) ![image](https://user-images.githubusercontent.com/10818702/70047845-2ca76880-15c9-11ea-951b-c14ac6e9d9e4.png)